### PR TITLE
Added endpoint to upload image

### PR DESCRIPTION
**Pull request by Lioncat2002**
The uploaded image is read as a python file object
which can then be passed on to the ML model 

Please have a look at the PR @z404

### CODE REVIEW

File: api/main.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+@app.get("/")
+def read_root():
+    return {"This is": "a test"}
+
+
+@app.post("/upload/")
+async def upload(file: UploadFile):
+    contents = await file.read()  # --> file is a standard python file object
```
Comment (z404) [inline api/main.py]: @Lioncat2002 Could you make sure that the file being accepted here is an image? If it isn't an image, the api should respond accordingly

File: api/main.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    # TODO: Add a function call here to the ML model which will accept the file object
+
+    # temporary code to write image on to the disk
+    with open("tmp/img.jpeg", "wb") as f:
```
Comment (z404) [inline api/main.py]: Even though this is temporary code, this line of code would throw an error on a computer which doesn't have a tmp file. You could write a fail safe by creating the folder if it doesn't exist.

**Review by z404 — CHANGES_REQUESTED** [review]

File: api/main.py (hunk 2 above)
Comment (Lioncat2002) [reply api/main.py]: Ok I will make the changes

**Comment by Lioncat2002** [comment]
God finally!

File: api/main.py (hunk 2 above)
Comment (Lioncat2002) [reply api/main.py]: Changes made
@z404

**Review by z404 — APPROVED** [review]
Looks good for now. @rithikjain you can review and merge

**Pull request merged**